src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -288,6 +288,12 @@ internal void ScheduleShutdownStandbyChannels()
         }
 
         public Task ShutdownChannelsAsync()
+        {
+            ShutdownChannels();
+            return Task.CompletedTask;
```

Comment:
Why do need to make a synchronous method for this? Isn't `GetFunctionMetadataAsync` an async function and should be able to call `ShutdownChannelsAsync`?